csharp/Samples/Microsoft.Spark.CSharp/DStreamSamples.cs

```diff
@@ -244,7 +244,7 @@ private static void DStreamReduceByKeyAndWindowSample()
         {
             count = 0;
 
-            const int bacthInterval = 2;
+            const long bacthInterval = 2000;
```

Comment:
consider renaming the variable to batchIntervalInMilliSeconds or something to that effect